### PR TITLE
Add SVG tutorial

### PR DESCRIPTION
**Pull request by daniellevass**
Hi All,

@mercebc and I have been working on an initial draft for an SVG tutorial. What do you think? 

Unsure about whether we should add a section for how to make path objects, or whether that might be too much for one tutorial?

The intention is to also have a part 2 to animate the emoji made using JavaScript.

Thank you 😄

### CODE REVIEW

File: js/lesson6-1/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+```xml
+<polygon points="0,0 500,0 500,500 0,500" fill="red" />
+```
+
+The `points` property looks super confusing, but really it's just 4 sets of coordinates (x,y). We start off at `0,0` which is top left, we then go right to `500,0`, down to `500,500` and then left to `0,500`. The polygon shape will automatically close itself by returning back to `0,0`.
+
+Another thing we need to remember is the `z index`. Shapes we draw later in the code, will get drawn on top of other shapes. So our red square will need to be drawn before our circle, to be drawn behind.
+
+Speak to your coach if your unsure about the z index, or how the points property works. For more information you could look at [http://www.w3schools.com/graphics/svg_polygon.asp](http://www.w3schools.com/graphics/svg_polygon.asp)
```
Comment (KimberleyCook) [inline js/lesson6-1/tutorial.md]: you're not your

File: js/lesson6-1/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+## Pause
+
+Spend some time making your emoji more awesome, you probably want some more circles for eyes. You could look at adding some more shapes for features, or accessories e.g. you could use a polygon to make a hat.
+
+![Imgur](http://i.imgur.com/Vxm3KVF.png)
+
+> here's some we made earlier :)
+
+There are also some other svg shapes that might be useful, such as lines, ellipses, and text. Work with your coach, and look at W3 Schools for more ideas: [http://www.w3schools.com/graphics/svg_intro.asp](http://www.w3schools.com/graphics/svg_intro.asp)
```
Comment (KimberleyCook) [inline js/lesson6-1/tutorial.md]: After this sentence could we add: 

"This ends our firts of 2 lessons on SVG, we hope you enjoyed it and learnt something. If you have some spare time how about going back through this tutorial and, by yourself, make some amendments. If there is something you did not understand or want to give us some feedback, please send us an email"

**Review by KimberleyCook — CHANGES_REQUESTED** [review]
Hey, added one lil comment. 

The final sentence comment is what is just at the end of all tutorials so needs to be there for consistency.

Thank you very much, this is really really great :D :D

**Comment by KimberleyCook** [comment]
@daniellevass @mercebc Just reminding you both of these little amends as I'd love to merge this in as it's great :)

**Comment by matyikriszta** [comment]
@mercebc @daniellevass what is the status of this issue? We would love to merge it in and add it to the tutorials 😄

**Comment by brunogirin** [comment]
This issue doesn't seem to have moved much for a year and I'd be interested in picking this up. However, I had in mind a more complex tutorial that covers multiple lessons, as I explained in [the SVG and canvas issue](https://github.com/codebar/tutorials/issues/100#issuecomment-443415227). Would that work?

**Pull request closed without merging**